docs/notebook/Tutorial.ipynb

```diff
@@ -757,6 +757,74 @@
     "print(conf.key)"
    ]
   },
+  {
```

Comment:
Looking at this diff is making it hard to know the structure is right.
The structure that makes sense to me:

1. Variable interpolation section contains a mention and an example of nesting interpolations.
2. Custom interpolation mentions that nested interpolations are supported, providing no example or a very short one.

Reply:
Question related to this, currently the section structure in the notebook is like this:
```
# Variable interpolation
### Config node interpolation
### Environment variable interpolation
## Custom interpolations
## Nested interpolations
```
I assume that the first two sub-sections should use `##` instead, am I correct?

Reply:
Changed in 6339fcd

Reply:
Probably, you can see what it looks like in the local browser by building the docs and opening the index locally.
Please do that do to make sure it all looks good (there are some details on how to do it at the bottom of [CONTRIBUTING.md](https://github.com/omry/omegaconf/blob/master/CONTRIBUTING.md).

Reply:
Checked, structure looks good to me.

I realized though that one example was too wide, at least on my computer (had to scroll to the right to see the full line of code). I fixed that in 5500100